source/includes/indexes/indexes.py

```diff
@@ -88,26 +88,56 @@
 collection.create_search_index(index)
 # end-create-search-index
 
+# start-create-vector-search-index
+
+from pymongo.operations import SearchIndexModel
+
+search_index_model = SearchIndexModel(
+  definition={
+    "fields": [
+      {
+        "type": "vector",
+        "numDimensions": <numberOfDimensions>,
+        "path": "<fieldToIndex>",
+        "similarity": "euclidean | cosine | dotProduct"
```

Comment:
S: change this presentation to make it more clear that the user needs to pick one
ie
```suggestion
        "similarity": "<select from euclidean, cosine, dotProduct>"
```